### ISSUE TITLE
Ling adrenals do not reset stamina.

### ISSUE THREAD

**Issue by HomocidlePancakes** [tgstation/tgstation#48341]
Got yeeted with a baton, used the ability aaaaaaaand I was still stunned.

**Comment by HomocidlePancakes** [comment]
Ok, I looked it up and the adrenalin only heals stamina, but does not reset it on use. Doesn't feel intended.

**Comment by nemvar** [comment]
This is *somewhat* intended behaviour since chemicals were intentionally not changed to mirror their old stun behaviour with stamina.
That doesn't mean this is a non-issue as it feels like there was very little consideration on how the new interactions should play out.

**Comment by kriskog** [comment]
Intentional. Adrenals will no longer always save your ass if you are already stunned,  but if used for example after someone gets a stun baton on you it will:
- give you stun resist, reducing the knockdown to 0.5s
- if used after you are knocked down, negates it
- lets you ignore damage (and stamina) slowdown.
- gives you a fairly decent stamina regen, makes you require more than two hits to stun.
- gives you a very fast 6 second speed boost.
- makes your screen shake a bit.
In addition, gives you some reduction towards harder stuns.

tl;dr: You are using them wrong. If someones harmbatoning your skull in, you are fucked.

**Comment by HomocidlePancakes** [comment]
Very well. I’ve always used adrenals to ambush people, sometimes getting stunned on purpose to land a hit while they are cuffing me. I’ll try to use them as you suggest, at least as cling. Tot version seems to be completely  outmatched by stimpack.

**Comment by kriskog** [comment]
Traitor adrenals are fairly similar to ling adrenals, with a longer lasting more moderate speed boost and heavier drawback. Stimpack isnt an implant, so it is so far left alone.
